### DISCUSSION TITLE
Formular Plugin

### DISCUSSION THREAD

**Discussion by falkei** [PapooSoftware/PapooCMS#14] [Q&A]
Woran kann es liegen, dass nach dem Versenden eines Formulars keine E-Mail versendet wird?
Die Nachricht taucht nur im Backend auf.
Ich habe bei den Formulardaten eine "E-Mail Adresse an die versendet wird" in der Form
Name;Empfänger E-Mail angegeben.

**Comment by ah-papoo** [comment]
Schauen Sie mal im Backend unter "System->Logfiles", nachdem Sie versucht haben ein Formular auszufüllen, dort sollten sich Logs zu den versendeten Mails finden, womit mal ein Problem feststellen können sollte.

**Reply by falkei** [reply]
Dies sind 3 Versuche mit verschiedenen Maileinstellungen ein Formular zu verschicken:
1. wie Systemeinstellungen
2. Benutzerdef. SMPT
3. ohne SMPT

27.02.2023 - 13:46:43; 2003:c3:730:3600:9c75:a03:83a1:5970; Fehler, Mail wurde nicht verschickt. Error: Mail Funktion konnte nicht initialisiert werden.; to: webmaster@drk-tornesch.de; cc: ; from: webmaster@drk-tornesch.de; from_text: ; from_name: ; subject: Anmeldung Kurs; sender: ; Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/110.0 

27.02.2023 - 13:51:58; 2003:c3:730:3600:9c75:a03:83a1:5970; Fehler, Mail wurde nicht verschickt. Error: Language string failed to load: tls; to: webmaster@drk-tornesch.de; cc: ; from: webmaster@drk-tornesch.de; from_text: ; from_name: ; subject: Anmeldung Kurs; sender: ; Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/110.0 

27.02.2023 - 14:00:03; 2003:c3:730:3600:9c75:a03:83a1:5970; Fehler, Mail wurde nicht verschickt. Error: Mail Funktion konnte nicht initialisiert werden.; to: webmaster@drk-tornesch.de; cc: ; from: webmaster@drk-tornesch.de; from_text: ; from_name: ; subject: Anmeldung Kurs; sender: ; Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/110.0

**Reply by ah-papoo** [reply]
Sehr wahrscheinlich, wird bei Ihrem Server/Mail-Server als Verbindungs-Prefix "ssl" anstatt das aktuell angegebene "tls" erwartet.

Ich habe bereits hierfür bereits ein Issue eröffnet (https://github.com/PapooSoftware/PapooCMS/issues/13) jedoch habe ich bisher selbst keine Zeit gefunden dieses Feature hinzuzufügen.

Entweder müssen Sie dieses selbst entwickeln, darauf warten, dass hier dieses hinzufügen oder alternativ, in der Datei "/lib/classes/mail_it_class.php" unter Zeile 210 die Prefix per Hand abändern, bis einer der oben genannten DInge passiert.

**Reply by falkei** [reply]
ok, danke
